cereal/log.capnp

```diff
@@ -1252,6 +1252,28 @@ struct LiveLocationKalman {
   }
 }
 
+
+struct LivePose {
+  # More info on reference frames:
+  # https://github.com/commaai/openpilot/tree/master/common/transformations
+  orientationNED @0 : Measurement32;
+  velocityDevice @1 : Measurement32;
+  accelerationDevice @2: Measurement32;
+  angularVelocityDevice @3 : Measurement32;
+
+  inputsOK @4 :Bool = true;
+  posenetOK @5 :Bool = true;
+  sensorsOK @6 :Bool = true;
+  deviceStable @7 :Bool = true;
+  filterState @8 : Measurement32;
+
+  struct Measurement32 {
+    value @0 : List(Float32);
```

Comment:
lists tend to be annoying (checking length, building the list, etc.). can we use xyz here instead?

Reply:
that complicates a little bit the migration on xx side, but its doable